src/components/Node.js

```diff
@@ -122,15 +129,7 @@ class PureNode extends PureComponent {
 PureNode.propTypes = {
   id: PropTypes.number.isRequired,
   node: PropTypes.object.isRequired,
-  schema: PropTypes.arrayOf(
-    PropTypes.shape({
-      name: PropTypes.string.isRequired,
-      attr: PropTypes.func.isRequired,
-      type: PropTypes.oneOf(['array', 'object', 'location']),
-      label: PropTypes.string,
-      showEmpty: PropTypes.bool
-    })
-  ).isRequired,
+  schema: PropTypes.func.isRequired,
```

Comment:
Maybe we could put the expected return type as a comment, copying the deleted `PropTypes.arrayOf & shape`.